### ISSUE TITLE
traffic_package_size is missing with `ByTrafficPackage` instance.

### ISSUE THREAD

**Issue by idy** [zenlayer/terraform-provider-zenlayercloud#1]
for `POSTPAID` and `ByTrafficPackage` type instance, the `traffic_page_size` should be able to set to 0. Now it reports an error:

```error
│ Error: Missing required argument
│ 
│   with zenlayercloud_instance.g1_net_servers,
│   on version.tf line 26, in resource "zenlayercloud_instance" "g1_net_servers":
│   26: resource "zenlayercloud_instance" "g1_net_servers" {
│ 
│ traffic_package_size is missing with `ByTrafficPackage` instance.
```

```
terraform {
  required_providers {
    zenlayercloud = {
      source = "zenlayer/zenlayercloud"
    }
  }
}

provider "zenlayercloud" {
  access_key_id       = ""
  access_key_password = ""
}

data "zenlayercloud_images" "ubuntu" {
  availability_zone = "SIN-Z"
  category          = "Ubuntu"
  image_type        = "PUBLIC_IMAGE"
  image_name_regex  = "LTS"
}

resource "zenlayercloud_subnet" "bug-report-1" {
  availability_zone = "SIN-Z"
  cidr_block        = "10.0.0.0/24"
}

resource "zenlayercloud_instance" "g1_net_servers" {
  availability_zone          = "SIN-Z"
  image_id                   = data.zenlayercloud_images.ubuntu.images[0].image_id
  instance_charge_type       = "POSTPAID"
  internet_charge_type       = "ByTrafficPackage"
  internet_max_bandwidth_out = 1000
  instance_type              = "s.1c1g"
  subnet_id                  = zenlayercloud_subnet.bug-report-1.id
  system_disk_size           = 30
  traffic_package_size       = 0
}


output "ubuntu" {
  value = data.zenlayercloud_images.ubuntu
}
```

-----

If I set the `traffic_package_size` to 1, the first time apply will create the instance. But it reports error if redo the apply:

```
resource "zenlayercloud_instance" "g1_net_servers" {
  availability_zone          = "SIN-Z"
  image_id                   = data.zenlayercloud_images.ubuntu.images[0].image_id
  instance_charge_type       = "POSTPAID"
  internet_charge_type       = "ByTrafficPackage"
  internet_max_bandwidth_out = 1000
  instance_type              = "s.1c1g"
  subnet_id                  = zenlayercloud_subnet.bug-report-1.id
  system_disk_size           = 30
  traffic_package_size       = 1
}
```

The result of first-time apply is OK:
```
Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

If redo the apply command without any changes to the config, it detects changes and reports error:

```
Terraform will perform the following actions:

  # zenlayercloud_instance.g1_net_servers will be updated in-place
  ~ resource "zenlayercloud_instance" "g1_net_servers" {
        id                         = "935976848972718006"
      ~ traffic_package_size       = 0 -> 1
        # (17 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

│ Error: [ZenlayerCloudSdkError] Code=INVALID_CHARGE_TYPE, Message=The operation is denied as the charge type `POSTPAID` is not support, RequestId=T187B0D23-9F18-4836-A1F8-57A719BA9213
│ 
│   with zenlayercloud_instance.g1_net_servers,
│   on version.tf line 26, in resource "zenlayercloud_instance" "g1_net_servers":
│   26: resource "zenlayercloud_instance" "g1_net_servers" {
│ 

```

**Comment by wolfgangzhu** [comment]
thanks for reporting this issue, we will fix this issue in next version

**Comment by wolfgangzhu** [comment]
resolved (v0.1.6)
